src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1363,14 +1388,16 @@ private async Task<StartTask> StartTaskIfNeeded(VirtualMachineConfiguration mach
             {
                 var commandLine = new StringBuilder();
                 commandLine.Append(@"/usr/bin/bash -c 'trap ""echo Error trapped; exit 0"" ERR; sudo touch tmp2.json && (sudo cp /etc/docker/daemon.json tmp1.json || sudo echo {} > tmp1.json) && sudo chmod a+w tmp?.json && if fgrep ""$(dirname ""$(dirname ""$AZ_BATCH_NODE_ROOT_DIR"")"")/docker"" tmp1.json; then echo grep ""found docker path""; elif [ $? -eq 1 ]; then ");
+
                 commandLine.Append(machineConfiguration.NodeAgentSkuId switch
                 {
                     var s when s.StartsWith("batch.node.ubuntu ") => "sudo apt-get install -y jq",
```

Comment:
You know that Ubuntu always has wget installed, or is this line missing wget?

Reply:
yes Ubuntu always has wget installed